src/components/Graph.react.js

```diff
@@ -622,7 +622,7 @@ const graphDefaultProps = {
     relayoutData: null,
     extendData: null,
     restyleData: null,
-    figure: {data: [], layout: {}},
```

Comment:
FWIW it seems to me nested defaults like this are a bit dangerous, as providing some other part of the `figure` prop will cause the rest of the defaults to be dropped - first mentioned in https://github.com/plotly/dash-core-components/issues/514. It's OK here because these defaults behave the same as the attributes being missing entirely, but by the same token is there actually a benefit to including any of them? Could we just say `figure: {}`?

Reply:
I don't mind either way, I was just following the current pattern :)

Can we get this one merged in and follow up in a different issue/PR?